### PR TITLE
Add codespell: config, workflow (to alert when new typos added) and get typos fixed

### PR DESCRIPTION
**Pull request by yarikoptic**
feel free to ignore/close

### CODE REVIEW

**Comment by estesp** [comment]
I'm fine adding this; definitely seems reasonable, but will need all 4 commits edited to add `Signed-off-by:` to pass CI. Thanks!

**Comment by yarikoptic** [comment]
force pushed signed off

File: .codespellrc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+[codespell]
+skip = .git,*.pdf,*.svg,go.sum,.codespellrc,vendor
+check-hidden = true
+# ignore-regex =
```
Comment (estesp) [inline .codespellrc:4]: git whitespace checks complaining about an extra space after the `=` on line 4.

You should be able to verify with: `git show --check 045a8a04e6354ac102b4ad8a46857e5c8d988edd`

Comment (yarikoptic) [reply .codespellrc]: thanks, fixed, rewritten (why to breed commits) and force pushed after re-signing-off

**Review by estesp — APPROVED** [review]

**Pull request merged**